Journal of Extension.js

```diff
@@ -0,0 +1,360 @@
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { 
+	return FW.doWeb(doc, url); }
+
+
+
+FW.Scraper({
+  itemType			:'journalArticle',
+  detect			:FW.Url().match(/\w+\d+(\.php|\.shtml)$/),
+  title				:FW.Xpath('//div[@class="DIV-title"]').text().trim(),
+  creators			:FW.Xpath('//div[@class="DIV-author"]/p/b|//div[@class="DIV-author"]/p/strong').text().cleanAuthor("author"),
+  publicationTitle	:"Journal of Extension",
+  abstractNote		:FW.Xpath('//div[@class="DIV-abstract"]/p').text().trim(),
+  date				:FW.Xpath('//p[@class="DIV-number"]|//p[@class="DIV-number screen_only"]').text().split(/\/\//),
+  volume			:FW.Xpath('//p[@class="DIV-number"]|//p[@class="DIV-number screen_only"]').text().match(/Volume\s(\d+)/, 1),
+  issue 			:FW.Xpath('//p[@class="DIV-number"]|//p[@class="DIV-number screen_only"]').text().match(/Number\s(\d+)/, 1),
+  tags				:FW.Xpath('//div[@class="DIV-keywords"]/a').text().split(/,/).trim(),
+  attachments		:	{	title:"Full Text PDF",
+  							type		:"application/pdf",
+  							snapshot	: false,
+  							url			:FW.Xpath('//div[@id="article-nav"]/ul/li[4]/a').key('href').text()
+  						},	
+  ISSN				:'1077-5315'
+  });
+  
+
+ 
+ FW.MultiScraper({
+  itemType		:'multiple',
+  detect		:FW.Url().match(/\d+\w+\/$/),
+  choices		:{
+  	titles		:FW.Xpath('//h4/a|//div[@class="row-fluid"]/div/p/a').text().trim(),
+  	url			:FW.Xpath('//h4/a|//div[@class="row-fluid"]/div/p/a').key("href")
+  }
+ });var testCases=[
+	{
+		"type": "web",
+		"url": "http://www.joe.org/joe/2012december/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://www.joe.org/joe/1983january/a3.php",
+		"items": [
+					   {
+			 "itemType": "journalArticle",
+			 "creators": [
+			   {
+				 "firstName": "Elizabeth Chase",
+				 "lastName": "Scott",
+				 "creatorType": "author",
+			   },
+			 ],
+			 "notes": [],
+			 "tags": [],
+			 "seeAlso": [],
+			 "attachments": [
+			   {
+				 "url": false,
+				 "title": "Full Text PDF",
+				 "type": "application/pdf",
+				 "snapshot": false,
+			   },
+			 ],
+			 "url": "http://www.joe.org/joe/1983january/a3.php",
+			 "abstractNote": "Abstract\n         Extension's role In educating consumers on food cooperatives is based on Michigan's experiences. These guidelines may provide you with an educational program to help fight inflation.",
+			 "date": "January 1983",
+			 "ISSN": "1077-5315",
+			 "issue": "1",
+			 "publicationTitle": "Journal of Extension",
+			 "title": "A Way To Fight Inflation: Food Cooperatives",
+			 "volume": "21",
+			 "libraryCatalog": "Journal of Extension",
+//		 -   "accessDate": "CURRENT_TIMESTAMP",
+			 "shortTitle": "A Way To Fight Inflation",
+		   }
+	]},
+		{
+		"type": "web",
+		"url": "http://www.joe.org/joe/1983january/a3.php",
```

Comment:
we don't need quite as many test cases -- one or two plus a multiple is plenty unless I'm overlooking the reason you're doing this.